apps/nextra/pages/zh/build/cli/install-cli/install-cli-linux.mdx

```diff
@@ -0,0 +1,80 @@
+---
+
```

Comment:
Nit: rm

Reply:
> Nit: rm

Since some titles can be displayed in Chinese, I believe the 'Title' field is still necessary.